app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/SessionDetailActivity.java

```diff
@@ -48,6 +54,37 @@ protected void onCreate(Bundle savedInstanceState) {
             }
         }
         replaceFragment(SessionDetailFragmentCreator.newBuilder(sessionId).build(), R.id.content_view);
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            getWindow().getSharedElementEnterTransition().addListener(new Transition.TransitionListener() {
```

Comment:
**[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: Class requires API level 19 (current min is 16): android.transition.Transition.TransitionListener [NewApi]